src/coreclr/vm/jitinterface.cpp

```diff
@@ -720,10 +720,10 @@ int CEEInfo::getStringLiteral (
 }
 
 size_t CEEInfo::printObjectDescription (
-        void*  handle,
-        char*  buffer,
-        size_t bufferSize,
-        size_t* pRequiredBufferSize)
+        CORINFO_OBJECT_HANDLE  handle,
```

Comment:
This should use `getObjectFromJitHandle` to convert `CORINFO_OBJECT_HANDLE` to `Object*`.